src/style.css

```diff
@@ -1,3 +1,67 @@
+html {
+  font-family: "Roboto", sans-serif;
+  --is-active-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.moyo_header {
+  display: flex;
+  padding-left: 50px;
+  padding-right: 50px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.home_link {
+  width: 40px;
+  height: 40px;
+}
+
+.nav_list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.nav_link {
+  font-family: Roboto, sans-serif;
+  display: flex;
+  cursor: pointer;
+  height: 60px;
+  color: #000;
+  font-size: 12px;
+  font-style: normal;
+  text-transform: uppercase;
+  flex-shrink: 0;
+  white-space: nowrap;
+  text-decoration: none;
+  align-items: center;
+  position: relative;
+}
+
+.nav_list li:nth-child(n+2) {
+  margin-left: 20px;
+}
+
+.nav_link:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  bottom: 0;
+  border-radius: 8px;
+}
+
+.is-active {
```

Comment:
`.is-active`, and `.nav_link:hover` have the same css rules. So use as a group
Example:
```
.selector1,
.selector2 {
  rule: value;
}
```